gateway/ln-gateway/src/actor.rs

```diff
@@ -111,10 +111,12 @@ impl GatewayActor {
 
     pub async fn stop_subscribing_htlcs(&mut self) -> Result<()> {
         if let Some(sender) = &self.sender {
-            sender
+            let _ = sender
```

Comment:
Comment explaining why please. Helps with head scratching in the future.

Reply:
Added